src/components/wizards/import-vm/ImportVmWizard.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Form, FormControl, Col, DropdownButton, MenuItem, FormGroup, ControlLabel, Wizard } from 'patternfly-react';
+
+export class ImportVmWizard extends React.Component {
+
+    state = {
+      activeStepIndex: 0
+    }
+  
+    wizardStepsImportVM = [
+      {
+        title: 'Basic Information',
+        render: () => <Form horizontal>
+          <FormGroup>
+            <Col sm={3} className="text-right">
+              <ControlLabel className="required-pf">Import Source Type</ControlLabel>
+            </Col>
+            <Col sm={4}>
+              <DropdownButton bsStyle="default" id="import-type-dropdown" title="--- Choose Import Source Type ---">
```

Comment:
This should be "Select" rather than "Choose" to be consistent with the new VM side of things.

Reply:
Changed to Select

Reply:
Changed to Select